pairing-conduct.md

```diff
@@ -1,7 +1,25 @@
+# Thoughts on Inclusivity and Community
+
+Our goal is to foster an inclusive community that is open to all, regardless of skill/experience level, gender, ethnicity, age, etc. As a member of this community, it is therefore your responsibility to do your part in making sure that other attendees feel welcomed and confortable.
+
+---
+
+### On Diversity
+
+The homogeneous makeup of the tech community is notoriously bad. Promoting diversity is a huge deal to us and is something we take very seriously. If you feel that we could be doing more in some way to be more inclusive and welcoming to under-represented and/or marginalized groups, please let us know immediately.
```

Comment:
at the end of that last sentence, can you add where to contact organizers (Meetup.com messages, slack, email) 

and a link to our CoC would be cool too :) thanks for doing this.